api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -53,16 +53,25 @@ service AccessListService {
   rpc ListAllAccessListMembers(ListAllAccessListMembersRequest) returns (ListAllAccessListMembersResponse);
   // GetAccessListMember returns the specified access list member resource.
   rpc GetAccessListMember(GetAccessListMemberRequest) returns (Member);
+  // GetStaticAccessListMember returns the specified access_list_member resource. If returns error
+  // if the target access_list is not of type static.
+  rpc GetStaticAccessListMember(GetStaticAccessListMemberRequest) returns (GetStaticAccessListMemberResponse);
```

Comment:
Could you add a brief overview  What is a difference from static no static endpoint for members monuments ?  If someone see this design for first time it will be confused why the `UpsertAccessListMember` was splited to `UpsertAccessListMember` and `UpsertStaticAccessListMember` and why the Upset can just upsert static access list member

Namely the API comment should explain  what problem this kind of API share  it trying to solve.  


nit: Can we group those endpoint together ?